### PR TITLE
Add `-nobattleye` launch option to docs for GTA V

### PR DESCRIPTION
**Pull request by JoshuaBrest**
As Rockstar Games added BattlEye to their game, the `-nobattleye` option is required for both online and offline play.

### CODE REVIEW

File: src/game-support/gta-5.md [hunk 1]
```diff
@@ -5,19 +5,24 @@
...
+> [!NOTE]
+> As of 2024, Rockstar Games added BattlEye anti-cheat software. Online play will not work on many official and unofficial servers.
 
 > [!WARNING]
 > The Rockstar Games Launcher will fail with the `The Rockstar Games Launcher failed to initialize!` error when `DXVK` is turned `off` in your `Bottle Configuration`. `DXVK` can turned `on` to workaround the launcher error but you might experience visual gliches (invisible guns, green car windows, brightness and reflection artifacts, etc) if you do so. For a glichless experience we suggest to keep `DXVK` turned `off` and follow the following setup to workaround the launcher issue instead.
 
 ## Setup
 
 - Install in Steam as normal.
+- Add the `-nobattleye` launch option to the game.
```
Comment (hahayupgit) [inline src/game-support/gta-5.md:19]: Clarify if this means Steam -> Game -> Properties -> Launch options

or Whisky launch options (I'm assuming Steam, just wanting to make sure)

File: src/game-support/gta-5.md [hunk 2]
```diff
@@ -5,19 +5,24 @@
...
 - In Whisky open "Bottle Configuration" and:
   - If on, turn `DXVK` off.
   - Click on "Open Wine Configuration" and:
      - Under the "Application" tab add `Launcher.exe` (most likely from C:\Program Files\Rockstar Games\Grand Theft Auto V).
      - While `Launcher.exe` is selected in the "Application" tab, switch to the "Libraries" tab and add `d3d10core`, `d3d11`, `dxgi` selecting the "Native then Buildin" option for all.
-     - Click "Apply" and start the game from Steam as usual.
+     - Click "Apply" and start the game from Whisky as usual.
```
Comment (hahayupgit) [inline src/game-support/gta-5.md:25]: Why change this from Steam to Whisky? I'm just wondering

**Comment by JoshuaBrest** [comment]
> You can’t play online unless you wanna get kicked/banned.

I believe community servers can still disable it.

**Comment by hahayupgit** [comment]
https://support.rockstargames.com/articles/33490543992467/Grand-Theft-Auto-Online-BattlEye-FAQ

> > Does BattlEye need to run when playing on a Community RP Server?
> > Official Rockstar Community (RP) Servers will not require BattlEye to play. Community Server launchers will disable BattlEye as part of their launch processes.

**Comment by JoshuaBrest** [comment]
> That should be added to the notes as regular GTA: Online _won’t_ work.

```diff
+ > [!NOTE]
+ > As of 2024, Rockstar Games added BattlEye anti-cheat software. Online play will not work on many official and unofficial servers.
```

File: src/game-support/gta-5.md (hunk 2 above)
Comment (JoshuaBrest) [reply src/game-support/gta-5.md:25]: cause then they add the launch option from whisky i think.

**Review by hahayupgit — APPROVED** [review]

**Pull request merged**